pkg/pillar/types/volumetypes.go

```diff
@@ -245,3 +289,75 @@ func (status VolumeRefStatus) IsContainer() bool {
 	}
 	return false
 }
+
+// LogCreate :
+func (status VolumeRefStatus) LogCreate() {
+	logObject := base.NewLogObject(base.VolumeRefStatusLogType, status.DisplayName,
+		status.VolumeID, status.LogKey())
+	if logObject == nil {
+		return
+	}
+	logObject.CloneAndAddField("refcount-int64", status.RefCount).
+		AddField("generation-counter-int64", status.GenerationCounter).
+		AddField("state", status.State).
```

Comment:
This should be status.State.String() to get more friendly logs, but that is an issue across the board so we can do it separately.